playwright-e2e/pages/dsm/home-page.ts

```diff
@@ -1,13 +1,63 @@
 import { expect, Locator, Page } from '@playwright/test';
+import { HomePageInterface } from 'pages/page-interface';
+import { DSMPageBase } from './page-base';
+import * as auth from 'authentication/auth-dsm';
+import Select from 'lib/widget/select';
+import { Navigation } from 'lib/component/dsm/navigation/navigation';
+import { StudyNav } from 'lib/component/dsm/navigation/enums/studyNav.enum';
+import ParticipantListPage from 'pages/dsm/participantList-page';
+import ParticipantPage from 'pages/dsm/participant-page';
 
 enum Titles {
   WELCOME = 'Welcome to the DDP Study Management System',
   SELECTED_STUDY = 'You have selected the ',
   STUDY = ' study.'
 }
 
-export default class HomePage {
-  constructor(private readonly page: Page) {}
+export default class HomePage extends DSMPageBase implements HomePageInterface {
+  private readonly navigation: Navigation;
+  private readonly participantList: ParticipantListPage;
+
+  constructor(page: Page) {
+    super(page);
+    this.navigation = new Navigation(page);
+    this.participantList = new ParticipantListPage(page);
+  }
+
+  /**
+   * Log into DSM application
+   * @param opts
+   */
+  async logIn(opts: { email?: string; password?: string; waitForNavigation?: boolean } = {}) {
+    await auth.login(this.page);
```

Comment:
I think when you are on this page, you are already logged in, therefore I don't think this method is useful here. What do you think about removing it? 
Please correct me if I'm wrong.

Usually, we just call the `login() `method from `beforeEach`. 
It would be good if we considered later adding **Login Page Object Model**, which would handle all login-related things